frontend/src/metabase/collections/components/CollectionContent/CollectionContentView.tsx

```diff
@@ -228,9 +228,11 @@ export const CollectionContentView = ({
     deleteBookmark(collectionId.toString(), "collection");
   };
 
-  const canUpload = uploadsEnabled && canUploadToDb && collection.can_write;
+  const canCreateUpload = canCreateUploadInDb && collection.can_write;
```

Comment:
The merge conflict was just this line, and was caused by the fact there's no trash collection in 50, but there is in master. See this:
https://github.com/metabase/metabase/pull/43026/files#diff-5dc398986435b54050d058bf164d7d7a5c3def1d9e66356cbfa53ecc8bcdb034R206-R209